### DISCUSSION TITLE
many to many soft delete

### DISCUSSION THREAD

**Discussion by world56** [prisma/prisma#13917] [Q&A]
<img width="342" alt="image" src="https://user-images.githubusercontent.com/37354312/174693718-f5b90747-d116-48eb-be8f-0daae7d27a5f.png">

<img width="448" alt="image" src="https://user-images.githubusercontent.com/37354312/174623011-d115b1ff-6c45-4ee7-a139-934ac295bd35.png">

How to achieve?
Thank you everyone🙏

**Comment by austincrim** [comment]
Hey @world56 👋,

Could you elaborate more on your question? I'm unsure what you're asking.

You mentioned soft delete, so you might be interested in [this docs page](https://www.prisma.io/docs/concepts/components/prisma-client/middleware/soft-delete-middleware).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
